examples/rust/minimal_serve/README.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
is there a way to preview this example? doesn't show up on the examples preview page and I'm fairly sure it's supposed to.

Reply:
It's not supposed to (you have to manually add it to the list of examples that get picked up).

It's just to show how to use the `serve` API, it doesn't make sense as a website example.